corehq/apps/app_manager/models.py

```diff
@@ -5723,6 +5735,73 @@ def create_suite(self, build_profile_id=None):
     def create_media_suite(self, build_profile_id=None):
         return MediaSuiteGenerator(self, build_profile_id).generate_suite()
 
+    @memoized
+    def get_practice_user_id(self, build_profile_id=None):
+        # returns app or build profile specific practice_mobile_worker_id
+        if build_profile_id:
+            build_spec = self.build_profiles[build_profile_id]
+            return build_spec.practice_mobile_worker_id
+        else:
+            return self.practice_mobile_worker_id
+
+    @property
+    @memoized
+    def enable_practice_users(self):
+        return (
+            self.supports_practice_users and
+            domain_has_privilege(self.domain, privileges.PRACTICE_MOBILE_WORKERS)
```

Comment:
There was a suite test that was failing which was hard to fix when I had it that way. And none of the suite code checks privileges, it felt like suite generation should take app doc and work without checking accounting requirements. Accounting requirements are/should be considered when app is edited